src/main/java/seedu/address/storage/CentralManager.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.storage;
+
+import java.util.Objects;
+
+import seedu.address.model.CustomerManager;
+import seedu.address.model.DriverManager;
+import seedu.address.model.task.TaskManager;
+
+/**
+ * A wrapper manager that consists of all the managers used for saving purpose.
+ */
+public class CentralManager {
```

Comment:
Do we store `IdManager` in this `CentralManager` too?

Reply:
We should, ill make the changes tonight

Reply:
solved